polymer.externs.js

```diff
@@ -1162,3 +1162,38 @@ Polymer.RenderStatus.whenReady = function(cb) {}
  * @param {...*} args The function arguments.
  */
 Polymer.RenderStatus.afterNextRender = function(element, fn, args) {}
+
+Polymer.AppLayout;
+
+/**
+ * @constructor
+ * @extends Object
+ */
+Polymer.AppLayout.LocalDomWithBackground = function(){};
+/**
+ * @type HTMLElement
+ */
+Polymer.AppLayout.LocalDomWithBackground.prototype.backgroundFrontLayer;
+/**
+ * @type HTMLElement
+ */
+Polymer.AppLayout.LocalDomWithBackground.prototype.backgroundRearLayer;
+/**
+ * @type HTMLElement
+ */
+Polymer.AppLayout.LocalDomWithBackground.prototype.background;
+
+/**
+ * @constructor
+ * @extends PolymerElement
+ */
+Polymer.AppLayout.ElementWithBackground = function(){};
+
+Polymer.AppLayout.ElementWithBackground.prototype = {
+  /** @type Polymer.AppLayout.LocalDomWithBackground */
```

Comment:
I'm not sure the compiler will be happy with this because you're overriding the type of the base class `$` property. Have you tried sending this through the compiler yet?